packages/apps/reputation-oracle/server/src/common/config/auth-config.service.ts

```diff
@@ -61,4 +61,11 @@ export class AuthConfigService {
       86400,
     );
   }
+
+  /**
+   * Human APP email.
+   */
+  get humanAppEmail(): string {
+    return this.configService.getOrThrow<string>('HUMAN_APP_EMAIL');
```

Comment:
Would it be useful to add email validation for this env variable to `envValidator` we have?